packages/deployer/test/sample-project/contracts/modules/AModule.sol

```diff
@@ -0,0 +1,14 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "../mixins/GenIMCMixin.sol";
+import "./SomeModule.sol";
+
+contract AModule is GenIMCMixin {
```

Comment:
Shouldn't there be a GenIMCMixin per network? If so, how does AModule know which one to extend?